ts/ui/lazy/LazyHandler.ts

```diff
@@ -584,6 +664,21 @@ B extends MathDocumentConstructor<HTMLDocument<N, T, D>>>(
       return items;
     }
 
+    /**
+     * @override
+     */
+    public render() {
+      //
+      // Get the containers whose content should always be typeset
+      //
+      const always = this.options.lazyAlwaysTypeset;
+      this.lazyAlwaysContainers = !always ? null :
```

Comment:
The `lazyAlwaysTypeset` option is not a boolean, but either a DOM element, a string used to select a DOM element, or  an array of those.  So it can't be the other way around (if I understand your comment).  Note that in the next line is

```
this.adaptor.getElements(Array.isArray(always) ? always : [always], this.document);
```

so `always` is being used to identify the elements to force to be always typeset (not lazy typeset).